panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,25 +59,44 @@
         </app-toolbar>
       </app-header>
 
+      <app-localstorage-document
```

Comment:
Why did you add all the other app-localstorage-document elements as children of this one?